cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/ExecutionTaskTracker.java

```diff
@@ -221,24 +249,24 @@ public int numPendingReplicaDeletion() {
    */
   public void clear() {
     // Clear dead tasks.
-    for (BalancingAction balancingAction : BalancingAction.values()) {
+    for (BalancingAction balancingAction : BalancingAction.cachedValues()) {
```

Comment:
Would each `_*Tasks` map be cleared just with a single clear call per map rather than clearing each of the sets in the map?

Reply:
This is to avoid recreating the sets. If the map is cleared, methods such as abortingTasksFor() will return null.